openproblems/tasks/chromatin_potential/datasets/__init__.py

```diff
@@ -0,0 +1 @@
+from .scicar_chrom import scicar_mouse_kidney
```

Comment:
Can we call this `.scicar` rather than `.scicar_chrom`?

Reply:
Yes, I'll fix it as .scicar.